section2/reflection.md

```diff
@@ -1,15 +1,26 @@
 
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+* At first it gave me a bit of anxiety because I was like "Oh no I'm supposed to know this!" But then I take a breath and remind myself I am capable and I am learning, and do the necessary research to check I am following the proper directions. This supported my learning by proving I am capable of figuring things out and that panic is not necessary.
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
+* A conditional statement says what to do in a given situation. A daily life conditional could be "if the alarm goes off, then I wake up, else I press snooze." An example of where a conditional is probably used in the Peloton app is "if a user takes a specific instructor's class, then recommend other classes by that same instructor."
 
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+* `else if` allows multiple conditions to be chained to an `if` statement. This allows for different actions to be performed for different decisions.
+* First the `if` statement specifies a block of code to be executed if the first condition is true. If it does not evaluate to be true, it will go to the next `else if` statement to evaluate this new condition. If this new condition evaluates to be true, there will be specific code executed. If it does not evaluate to be true, it will continue on to the next `if else` statement, testing that new condition and evaluating it to be true or false. There will be specific code executed for when an `if else` condition is met. If all the `if else` statements evaluate to not true, `else` allows you to specify a block of code to be executed if none of the conditions are met.
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+* Logical operators - `&&` is available to check that two conditions are met. `||` is available to check that 1 of 2 conditions are met before running a block of code.
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?
+* Is there a limit to the number of `else if` statements that can be chained to an `if` statement? If there is an `if` statement with multiple `else if` statements, does it still need to end in an `else`?
```

Comment:
Great question!

The `else` is never required.

You can have as many `else if`s as your heart desires.